etc/workloads/service-architecture/PrimaryRequiredReads.yml

```diff
@@ -0,0 +1,67 @@
+SchemaVersion: 2018-07-01
+Pool:
+    QueryOptions:
+        maxPoolSize: 1000
```

Comment:
1. why 1k here? seems like you should only need 100ish. Nothing "wrong" with more connections than needed just wondering if there was a reason for the order-of-magnitude difference. If there is a reason, perhaps a comment would help.
2. Do you also want a min pool size to do all the connection acquisition up front?

Reply:
1. It's mostly about that order of magnitude difference. It needs to be set higher because the default is 100. So an order of magnitude is reasonable.
2. I'm not sure that actually forces connection. It's just the amount of pool entries that can be floating about IIRC.